lib/view/HandsOn.dart

```diff
@@ -1,7 +1,20 @@
+import 'package:audioplayers/audioplayers.dart';
 import 'package:flutter/material.dart';
 
 class HandsOn extends StatelessWidget {
-  const HandsOn({Key? key}) : super(key: key);
+  HandsOn({Key? key}) : super(key: key);
+  final List<Map<String, String>> songList = [
+    {"初恋" : "firstlove"},
+    {"レモン" : "lemon"},
+    {"マリーゴールド" : "marigold"},
+    {"チェーンソーマン" : "chainsawman"},
```

Comment:
```suggestion
    {"KICKBACK" : "chainsawman"},
```
細かいですが、チェーンソーマンではなく、チェンソーマンですね